src/main/java/dev/ebyrdeu/movie_jakarta/resource/MovieResource.java

```diff
@@ -4,78 +4,65 @@
 import dev.ebyrdeu.movie_jakarta.dto.Movies;
 import dev.ebyrdeu.movie_jakarta.entity.Movie;
 import dev.ebyrdeu.movie_jakarta.repository.MovieRepository;
+import dev.ebyrdeu.movie_jakarta.service.MovieService;
+import jakarta.el.CompositeELResolver;
 import jakarta.inject.Inject;
 import jakarta.validation.Valid;
 import jakarta.validation.constraints.NotNull;
 import jakarta.ws.rs.*;
-import jakarta.ws.rs.core.EntityTag;
 import jakarta.ws.rs.core.MediaType;
 import jakarta.ws.rs.core.Response;
 
 import java.net.URI;
 import java.util.Optional;
 import java.util.UUID;
-import java.util.stream.Collectors;
 
 @Path("/movies")
 public class MovieResource {
 
-    private MovieRepository movieRepository;
+    MovieService movieService;
 
-    public MovieResource() {
+    public MovieResource(){
     }
-
     @Inject
-    public MovieResource(MovieRepository movieRepository) {
-        this.movieRepository = movieRepository;
+    public MovieResource(MovieService movieService) {
```

Comment:
Try to use instead:
@Inject
public MovieResource(MovieService movieRepository) {
        this.movieService = movieRepository;
    }